src/index.js

```diff
@@ -1,18 +1,21 @@
-import React from 'react'
+import React, { Suspense } from 'react'
 import ReactDOM from 'react-dom'
 import {BrowserRouter} from 'react-router-dom'
 import { ThemeProvider } from 'styled-components'
 import './index.css'
 import App from './components/App'
 import * as serviceWorker from './serviceWorker'
 import theme from './styles/theme'
+import './i18n/i18n'
 
 const rootComponent = (
-    <BrowserRouter>
-        <ThemeProvider theme={theme}>
-            <App/>
-        </ThemeProvider>
-    </BrowserRouter>
+    <Suspense fallback={null}>
```

Comment:
why using a Suspense here?

Reply:
For i18n. Because an error was throwned. I'll try to remove it